### PR TITLE
bin: Add msg dump utils

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang PTAL

### CODE REVIEW

File: src/bin/dump-msg.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    let args: Vec<String> = env::args().collect();
+    let program = args[0].clone();
+    let mut opts = Options::new();
+    opts.optopt("", "db", "set rocksdb path, required", "");
+    opts.optopt("r",
+                "region-id",
+                "set the region id",
+                "required when getting raft message");
+    opts.optflag("h", "help", "print this help menu");
+    opts.optopt("i", "index", "set the raft log index", "");
```
Comment (siddontang) [inline src/bin/dump-msg.rs]: only log? 
Region meta info is needed too, I think.

**Comment by siddontang** [comment]
LGTM

I think we can add more dump later.

**Comment by BusyJay** [comment]
@ngaut PTAL

File: Cargo.toml [hunk 2]
```diff
@@ -17,6 +17,9 @@ name = "tikv-server"
 name = "bench-tikv"
 path = "benches/bin/bench-tikv.rs"
 
+[[bin]]
+name = "dump-msg"
```
Comment (ngaut) [inline Cargo.toml]: The name is not  appropriate.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**